ras_backstage/__init__.py

```diff
@@ -70,6 +70,7 @@
 from ras_backstage.resources.sign_in.sign_in import SignInV2  # NOQA # pylint: disable=wrong-import-position
 from ras_backstage.resources.survey.get_survey_list import GetSurveyList  # NOQA # pylint: disable=wrong-import-position
 from ras_backstage.resources.survey.get_survey_by_short_name import GetSurveyByShortName  # NOQA # pylint: disable=wrong-import-position
+from ras_backstage.resources.secure_messaging.get_thread import GetThread  # NOQA # pylint: disable=wrong-import-position
```

Comment:
Can this go in with the other secure_message module imports.

Reply:
Can we do from ras_backstage import resources? @JamesGardiner

Reply:
To access this class you would then have to do `resources.get_thread.GetThread` which is overly verbose. Better to put it in the right import position.

Reply:
done it.